zulipterminal/ui_tools/views.py

```diff
@@ -1603,7 +1611,7 @@ def __init__(
         full_raw_message_keys = "[{}]".format(
             ", ".join(map(str, display_keys_for_command("FULL_RAW_MESSAGE")))
         )
-        msg_info = [
+        msg_info: PopUpViewTableContent = [
```

Comment:
It doesn't work otherwise since `PopUpViewTableContent` also has types which doesn't support append so need an explicit assertion. 
I can't not explicitly type it as `PopUpViewTableContent` since Mypy assumes it to be the type of the first data assigned to it.